bin/variables/load-prod-environment-variables.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -eu
+
+echo 'export ENVIRONMENT=prod'
+
+# urls
+echo "export SERVICE_AUTH_PROVIDER_API_BASE_URL=http://rpe-service-auth-provider-prod.service.core-compute-prod.internal"
+echo "export IDAM_API_BASE_URL=https://idam-api.prod.platform.hmcts.net"
```

Comment:
i think should be https://idam-api.platform.hmcts.net

Reply:
👍